src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -85,6 +85,8 @@ interface NavBarPropsType {
 }
 
 const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
+  const notificationSound = "/sounds/notification.m4a";
+  
```

Comment:
> @0xdevalias:
>
> Trying this locally, I see the error: `Cannot find module 'src/assets/notification.mp3' or its corresponding type declarations.`
> 
> Googling for that lead me to <https://stackoverflow.com/questions/56782452/how-to-fix-module-not-found-for-audio-files-using-file-loader-images-css-an>
> 
> The solution in <https://stackoverflow.com/questions/56782452/how-to-fix-module-not-found-for-audio-files-using-file-loader-images-css-an/60235615#60235615> sounds like it might do the trick
> <sub>[View in Slack](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1613460519111500)</sub>

---

> @0xdevalias:
>
> In `src/react-app-env.d.ts` i added:
> 
> ```
> declare module "*.mp3" {
>   const src: string;
>   export default src;
> }
> ```
>
> Then in my react component file, I used: `import notificationSound from "assets/notification.mp3";` and didn’t see any errors
> 
> <sub>[View in Slack](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1613460738111800)</sub>